spec/protocol-spec/experiments/agreement-violation.txt

```diff
@@ -0,0 +1,202 @@
+[State 27]
```

Comment:
This example is outdated. I propose to remove it from this PR.

Reply:
ok